packages/server/src/auth/memberships.ts

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: Copyright Orangebot, Inc. and Medplum contributors
+// SPDX-License-Identifier: Apache-2.0
+import { Operator } from '@medplum/core';
+import type { ProjectMembership, Reference, User } from '@medplum/fhirtypes';
+import type { Request, Response } from 'express';
+import { getAuthenticatedContext } from '../context';
+import { getSystemRepo } from '../fhir/repo';
+
+export async function membershipsHandler(req: Request, res: Response): Promise<void> {
+  const systemRepo = getSystemRepo();
+  const { authState } = getAuthenticatedContext();
+  const { login } = authState;
+  const userRef = login.user as Reference<User>;
+
+  const memberships = await systemRepo.searchResources<ProjectMembership>({
+    resourceType: 'ProjectMembership',
+    filters: [
+      {
+        code: 'user',
+        operator: Operator.EQUALS,
+        value: userRef.reference as string,
+      },
+    ],
+  });
+
+  const result = {
+    memberships: memberships.map((membership) => ({
```

Comment:
Assuming we don't limit to `authState.project`, this seems like a lot of info to be returning about memberships to other projects?